org.hydrogenmusic.Hydrogen.json

```diff
@@ -95,6 +160,12 @@
     "shared-modules/linux-audio/liblo.json",
     {
       "name": "vamp-plugin-sdk",
+      "buildsystem": "simple",
```

Comment:
what wrong with just using `"no-parallel-make": true`

Reply:
Nothing, I just wasn't aware that option existed.

Reply:
built locally so changed it